x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/utils/StringUtils.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.eql.utils;
+
+import org.elasticsearch.xpack.ql.expression.predicate.regex.LikePattern;
+
+public class StringUtils {
```

Comment:
Make the class `abstract` or better yet final + private constructor to prevent instantiation/inheritance.